control_plane/src/background_process.rs

```diff
@@ -252,6 +257,28 @@ fn fill_aws_secrets_vars(mut cmd: &mut Command) -> &mut Command {
     cmd
 }
 
+/// Add a `pre_exec` to the cmd that, inbetween fork() and exec(),
+/// 1. Claims a pidfile with a fcntl lock on it and
+/// 2. Sets up the pidfile's file descriptor so that it (and the lock)
+///    will remain held until the cmd exits.
+unsafe fn pre_exec_create_pidfile<P>(cmd: &mut Command, path: P) -> &mut Command
```

Comment:
Consider this as a NIT since it's in the control_plane code only

> unsafe

I wonder, is there's anything particular unsafe in the combination of the command and the path?
Feels like we could move unsafe block below, around `pre_exec`?

I also like SAFETY comments, both on the invocation place and the function (since it's unsafe, would be good to know about its invariants)

Reply:
Done.